### PR TITLE
filter upcoming text response to return only relevent events

### LINKED ISSUE #375: text command upcoming should only show events appropriate for viewer
Currently the text message 'upcoming' to the system returns all future events regardless of whether the person is not eligible for working the event.
Example a CERT person will see events which should only be shown to MRC personnel.

### PR DESCRIPTION
**Pull request by sa77**
resolves #375

### CODE REVIEW

File: app/services/msg/upcoming.rb [hunk 1]
```diff
@@ -4,25 +4,27 @@ def respond
     events = Event.upcoming.limit(10)
     response = ""
     events.each do |event|
```
Comment (kgf) [inline app/services/msg/upcoming.rb]: What if instead we changed this line to `@person.department.events` ? Then we don't need the conditional later ?

**Comment by kgf** [comment]
@sa77 Thanks for your help. Info requested.

File: app/services/msg/upcoming.rb (hunk 1 above)
Comment (sa77) [reply app/services/msg/upcoming.rb]: nice .. that works better

**Comment by sa77** [comment]
@kgf PR updated

**Comment by kgf** [comment]
@sa77 Thanks for all your fine work ! The texting section is much better now.
FYI, the emojis are not working _quite_ as well as I'd hoped ( although people really like having the info ), because the colors don't show properly. On a text message, the background color of the bubble varies from person to person. So the color of the emojii changes too !
Anyway, thanks a lot again.
I hope to see more PRs from you in the future.
Let me know if there is another area thta interests you
Regards,
Kevin

**Comment by sa77** [comment]
@kgf regarding SMS emojis, i think the handsets behave differently (maybe with the bubble background, user's personal settings, mobile OS, etc). 
Your project management, quick response to comments, PR review skills are amazing. Keep up the good work. I look forward to contributing more to the codebase. Cheers !

**Comment by kgf** [comment]
Thanks, I appreciate your comments !

**Pull request merged**